src/components/LogoMaker/LogoIcon/LogoIcon.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+<div>
+  <v-container class="icon-select-component pt-12 mt-12" fluid>
```

Comment:
`icon-select-component` > `logo-icon`. Also remove dependency from bootstrap is possible

Reply:
what does fluid do?

Reply:
thats not bootsrap i am using vuetify

Reply:
fluid is to remove padding and margin